src/redux/books/books.js

```diff
@@ -1,18 +1,41 @@
-const ADDBOOK = 'bookstore/books/ADDBOOK';
-const REMOVEBOOK = 'bookstore/books/REMOVEBOOK';
+import { generate } from 'randomized-string';
+import { alphanumeric } from 'randomized-string/lib/types';
 
-const booksReducer = (state = {}, action) => {
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
+const initState = [{
+  id: generate({ charset: alphanumeric }),
+  title: 'The book of boba fet',
+  author: 'Lucas',
+  completed: false,
+},
+{
+  id: generate({ charset: alphanumeric }),
+  title: 'The mandalorian',
+  author: 'JJ',
+  completed: false,
```

Comment:
- [ ] Please remove all the hard-coded objects that you have added to display the book.